account_manager/src/validator/cli.rs

```diff
@@ -0,0 +1,317 @@
+use bls::PublicKey;
+pub use clap::Parser;
+use clap::Subcommand;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+use super::exit::DEFAULT_BEACON_NODE;
+
+#[derive(Parser, Clone, Deserialize, Serialize, Debug)]
+#[clap(about = "Provides commands for managing Eth2 validators.")]
+pub struct Validator {
+    #[clap(
+        long,
+        value_name = "VALIDATOR_DIRECTORY",
+        conflicts_with = "datadir",
+        help = "The path to search for validator directories. \
+                Defaults to ~/.lighthouse/{network}/validators"
+    )]
+    pub validator_dir: Option<PathBuf>,
+    #[clap(subcommand)]
+    pub subcommand: ValidatorSubcommand,
+}
+
+#[derive(Parser, Clone, Deserialize, Serialize, Debug)]
+#[clap(rename_all = "kebab-case")]
+pub enum ValidatorSubcommand {
+    Create(Create),
+    Exit(Exit),
+    Import(Import),
+    List(List),
+    Recover(Recover),
+    #[clap(subcommand)]
+    Modify(Modify),
+    #[clap(subcommand)]
+    SlashingProtection(SlashingProtection),
+}
+
+#[derive(Parser, Clone, Deserialize, Serialize, Debug)]
+#[clap(
+    about = "Creates new validators from an existing EIP-2386 wallet using the EIP-2333 HD key \
+            derivation scheme."
+)]
+pub struct Create {
+    #[clap(
+        long,
+        value_name = "WALLET_NAME",
+        help = "Use the wallet identified by this name"
+    )]
```

Comment:
Currently the flags are not sorted alphabetically. We can add `display_order = 0` to make it sorted, as in the validator client and database cli files.

```suggestion
        help = "Use the wallet identified by this name",
        display_order = 0
    )]
```

This applies to all flags in the `cli.rs` file